bpmn/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20.18-alpine3.19
```

Comment:
I would prefer to use an explicit tag otherwise we will switch to the next LTS (when we don't expect it).

Reply:
Makes total sense 👍 I just understood that you wanted to target the `lts` tag.

Do you want to target only the major version (e.g. `22-alpine3.21`) or including the minor version like it is done currently (e.g. `22.14-alpine3.21`)?

Reply:
> Makes total sense 👍 I just understood that you wanted to target the lts tag.

Sorry, I wasn't clear 😉 

> Do you want to target only the major version (e.g. 22-alpine3.21) or including the minor version like it is done currently (e.g. 22.14-alpine3.21)?

I think major + minor is good so `22.14-alpine3.21`

Reply:
Great, fixed now

Reply:
@ggrossetie Could I request a re-review?

Reply:
it looks good, thanks 👍🏻